tests/test_repeated_dependency_schema.py

```diff
@@ -0,0 +1,103 @@
+from fastapi import Depends, FastAPI, Header
+from starlette.status import HTTP_200_OK
+from starlette.testclient import TestClient
+
+app = FastAPI(title="Access", redoc_url=None)
```

Comment:
could remove title and redoc arguments here that slipped in from my project, sorry :)